### PR TITLE
package.json - fix main reference

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "libp2p-floodsub",
   "version": "0.3.0",
   "description": "libp2p-floodsub, also known as pubsub-flood or just dumbsub, this implementation of pubsub focused on delivering an API for Publish/Subscribe, but with no CastTree Forming (it just floods the network).",
-  "main": "lib/core/index.js",
+  "main": "src/core/index.js",
```
Comment (haadcode) [inline package.json]: I believe this should be `"main": "src/index.js"`.

**Comment by haadcode** [comment]
Actually, take that back. pointing to `lib/index.js` might be desired behaviour. @diasdavid ping. This is due to the ES5 transpiling, right?

**Comment by diasdavid** [comment]
yep, it should be lib, that is what gets published. See https://github.com/ipfs/community/blob/master/js-project-guidelines.md

File: package.json (hunk 1 above)
Comment (diasdavid) [inline package.json]: it should be "lib/index.js"

**Comment by kumavis** [comment]
oh! build-and-publish. gotcha. I think `"main": "dist/index.js" would be more clear, but may be difficult to change that at this point.

**Comment by dignifiedquire** [comment]
@kumavis it's not ideal, but we have a plan of moving to `main: 'src/index.js` for our modules. You can see details for this here: https://github.com/ipfs/js-ipfs/issues/429

**Comment by kumavis** [comment]
@dignifiedquire thanks! just got off a call with @diasdavid and he explained the situation in depth

**Pull request closed without merging**